pkg/minikube/constants/constants.go

```diff
@@ -144,6 +150,17 @@ const (
 	DefaultMountVersion  = "9p2000.u"
 )
 
+func GetKubernetesReleaseURL(binaryName, version string) string {
+	// TODO(r2d4): change this to official releases when the alpha controlplane commands are released.
+	// We are working with unreleased kubeadm changes at HEAD.
+	return fmt.Sprintf("https://storage.googleapis.com/minikube-builds/v1.7.3/%s", binaryName)
```

Comment:
Should this be v1.7.3?
Also does kubeadm bootstrapper respect the k8s version flags?